File: Tests/Oracle/corpus/sample.kt
enum class Colour {
    RED,
    GREEN,
}

class Widget(private val name: String) {
    init {
        require(name.isNotEmpty())
    }

    fun shout(): String {
        return name.uppercase()
    }

    fun describe(n: Int): String {
        return when (n) {
            0 -> "none"
            else -> "some"
        }
    }
}

fun total(values: List<Int>): Int {
    var sum = 0
    values.forEach {
        sum += it
    }
    try {
        return sum
    } catch (e: Exception) {
        return 0
    } finally {
        println("done")
    }
}

File: Tests/Oracle/corpus/sample.cs
namespace Demo
{
    enum Colour
    {
        Red,
        Green,
    }

    class Widget
    {
        private int[] xs = new int[] { 1, 2, 3 };

        public int Size
        {
            get { return xs.Length; }
        }

        public int Total()
        {
            var sum = 0;
            foreach (var x in xs)
            {
                sum += x;
            }
            return sum;
        }
    }
}
